src/server/scripts/Outland/zone_blades_edge_mountains.cpp

```diff
@@ -157,7 +157,7 @@ class npc_deaths_door_fell_cannon_target_bunny : public CreatureScript
                             me->SummonCreature(NPC_HOUND, 0, 0, 0, 0, TEMPSUMMON_TIMED_DESPAWN_OUT_OF_COMBAT, 10000);
                         else 
                             me->SummonCreature(NPC_FEL_IMP, 0, 0, 0, 0, TEMPSUMMON_TIMED_DESPAWN_OUT_OF_COMBAT, 10000);
-                            events.ScheduleEvent(EVENT_PARTY_TIMER, 3000);
+                        events.ScheduleEvent(EVENT_PARTY_TIMER, 3000);
```

Comment:
Note: this might be a bug. Here I'm just changing the indentation to make it clear that not all statements are part of the IF block. **I'm NOT changing the behaviour**.

Reply:
same here